### DISCUSSION TITLE
Request permission for host a mirror doc

### DISCUSSION THREAD

**Discussion by ThornbirdSitcom** [official-antistasi-community/A3-Antistasi#2609] [General]
Hey there! My squads pretty enjoy this and we sincerely appreciate for your work!
In order to make them fully understand how to play this mission, I as one of person in charge in my team, have translated your doc site into Chinese. Obviously if I publish it as website it could be the best, but a website is considered as public. I'm worried about the possible violation of any copyright, so I'm here to cordially asking for your permission. I'm promising here if you agree:
1. The website I will host is only a Chinese mirror site of Antistasi doc. Without your further permission, there will be no more advertisement or any expression that not derive from the original site.
2. We will only identify ourselves at the beginning of doc. It will be only a few sentences in order to avoid possible misunderstand. 
3. I will appreciate if you can provide us the changelog, for we can update the site much faster.
4. Every time before the mirror publish or update, we will ask for your check.
No matter it is agreed or not, I am, personally and sincerely, thank you again for your such a great project! Looking forward to your reply!

**Comment by Bob-Murphy** [comment]
Hey, first of all thank you for asking. That in itself is very much appreciated.
In general what you are proposing sounds completely fine.
But one thing, which doc site are you talking about?
Are you taking about the GitHub Wiki (which could be understood as a doc) or full Guide using GitHub pages https://official-antistasi-community.github.io/A3-Antistasi-Docs/index.html ?

1. Sounds good
2. Sounds good
3. If Wiki - there is no changelog
If GitHub Pages - you can see the changes in the correct repo (https://github.com/official-antistasi-community/A3-Antistasi-Docs)
4. I don't know how we should be able to check. Afterall we don't speak/read/write Chinese

Could you please provide me a link or a screenshot of how your version looks like?

**Reply by ThornbirdSitcom** [reply]
It is the guide on your official site https://official-antistasi-community.github.io/A3-Antistasi-Docs/index.html we are talking about. 
Here is a screenshot of our translation. As you can see, we retain exactly how it looks like but in Chinese. The final proofreading hasn't finished yet, so some of them might seem weird. We are adjusting them :)

![1](https://user-images.githubusercontent.com/121800450/210236789-68e8083e-861e-4c5a-895d-d312f8bef25a.png)

As for how to check, I'm quite sorry I can't find any more solution. Sorry for that... I will try my best if you ask...

**Comment by Bob-Murphy** [comment]
Impressive. That must have been quite a bit of work you did there!

There might be the option to even integrate that into the official Guide as it's build with Sphinx and Sphinx does offer multi language support.
If that indeed is possible and you would be willing to work directly with us, that of course would be the preferred way to publish it, as this could make it available to more people. 

We'll try to check that within the next few days and let you know here. I don't know how busy the people are that are mostly dealing with the guide.

**Reply by ThornbirdSitcom** [reply]
Good to hear I can help! Waiting for good news!
Sincerely thanks to your reply!
